docs/faq/network/blockchain-architecture.md

```diff
@@ -0,0 +1,103 @@
+
+The design intention is to ensure that validators have a **solid incentive to remain online** and participate in the consensus process, but without making the penalties so severe that minor issues could result in significant losses. This balance aims to encourage a secure and decentralized network.
+
+### What are epochs?
+
+An epoch in PoS is a fixed period during which slots occur. It is a larger time frame that **helps to organize the work of validators who propose and attest** to blocks. An epoch is comprised of 32 slots, which means an epoch lasts for about 6.4 minutes, given that each slot is about 12 seconds.
+
+Epochs provide several vital functions:
+
+- **Validator Shuffling**: At the start of each epoch, a random selection process determines the active validators and assigns them to slots. This is done to ensure that the system remains decentralized and that no single validator can predict far in advance when they will be selected.
+- **Rewards and Penaltie**s: At the end of each epoch, rewards and penalties are calculated for validators. Validators that correctly proposed and attested to blocks receive rewards, while those who behaved maliciously or were offline are penalized.
+- **Finality**: An epoch also plays a role in achieving finality. Finality refers to the point at which a block cannot be changed or removed from the blockchain. The finality is achieved in every epoch.
+
+### What are slots?
+
+A slot in PoS is a period within an epoch where a **randomly chosen validator can propose a new block** to the blockchain during a slot. The role of a slot includes:
+
+- **Block Proposal**: Each slot represents an opportunity for a validator to propose a new block. If the selected validator is online and behaves correctly, they will propose a block, which other validators will attest to.
+- **Attestations**: During each slot, validators who are not chosen to propose a block are expected to attest to the validity of the proposed block. These attestations are important for determining consensus and helping the network agree on the state of the blockchain.
+- **Missed Slots**: If a chosen validator is offline or fails to propose a block during their slot, the slot is skipped, and no new block is added to the chain for that slot.
+
+A justified slot is a block that has been **voted on and is a candidate for finalization**. Essentially, justifying a block is the step before finalizing it. A block is justified when it receives 2/3 of the voting weight from the active validators. Justified blocks can still be overwritten if a conflicting block receives more votes. However, once a justified block is finalized, meaning the network has reached the consensus of the proposed block, it cannot be changed.
+
+### What are the expected block and epoch times?
+
+The LUKSO network will run with an expected block time of 12 seconds with 12 seconds per slot. An epoch contains 32 slots.
```

Comment:
"with an expected block time of 12 seconds with 12 seconds per slot." => weird phrasing. Maybe we can remove the whole part after the first "12 seconds"?